nixos/modules/services/misc/klipper.nix

```
@@ -125,6 +125,9 @@ in
                 type = path;
                 description = lib.mdDoc "Path to firmware config which is generated using `klipper-genconf`";
               };
+              package = lib.mkPackageOptionMD pkgs "klipper-firmware" {
```

nonblocking nit: This will probably be fixed in a later treewide PR, but I'll just leave a comment about it here.

```suggestion
              package = lib.mkPackageOption pkgs "klipper-firmware" {
```